bugsnag/handlers.py

```diff
@@ -56,6 +56,13 @@ def emit(self, record):
 
         client = self.client or bugsnag.legacy.default_client
 
+        if 'exception' in options:
+            if isinstance(options['exception'], Exception):
+                client.notify(**options)
+                return
+            else:
+                options['meta_data']['exception'] = options.pop('exception')
```

Comment:
Potentially want a check here to ensure that none of the callbacks changed the type of the `meta_data` object and it can be subscripted.

Reply:
I've updated to drop the `options['exception']` in the case it's not a dict anymore.